cmd/kube-scheduler/app/server.go

```diff
@@ -282,6 +282,16 @@ func getRecorderFactory(cc *schedulerserverconfig.CompletedConfig) profile.Recor
 		return cc.EventBroadcaster.NewRecorder(name)
 	}
 }
+// startInformers starts the informers that the scheduler needed.
+func startInformers(factory informers.SharedInformerFactory,stopChan <- chan struct{}){
+	// Start Informers
+	go factory.Core().V1().Pods().Informer().Run(stopChan)
```

Comment:
I don't know whether this reduced set of informers is sufficient, but the `sharedInformerFactory.Start` function does more than just start goroutines:

https://github.com/kubernetes/kubernetes/blob/b1b93e30131986319f1d46a62771feb9c0f097ae/staging/src/k8s.io/client-go/informers/factory.go#L127

It first obtains a lock. Then it checks and sets `startedInformers` too.

Besides, doesn't the list of informers to start depend on the options provided?

Reply:
Also what was the full set of informers and why is this reduced set sufficient? i.e. what informers are being removed?

Reply:
+1 to @adtac. I don't think this is the right direction. No need to mention that the out-of-tree plugins can instantiate whatever XYZInformer they want.

Also my understanding is that there is wasting on starting unneeded informers: https://github.com/kubernetes/kubernetes/pull/93959#issuecomment-673569456